evennia/server/portal/telnet.py

```diff
@@ -50,6 +49,8 @@ def connectionMade(self):
         # when it reaches 0 the portal/server syncs their data
         self.handshakes = 8  # suppress-go-ahead, naws, ttype, mccp, mssp, msdp, gmcp, mxp
         self.init_session(self.protocol_name, client_address, self.factory.sessionhandler)
+        # change encoding to ENCODINGS[0] which reflects Telnet default encoding
+        self.protocol_flags["ENCODING"] = settings.ENCODINGS[0]
```

Comment:
Sorry for being a pest about it, but thinking about it, since `ENCODINGS` is controlled by the user it may be empty, maybe this would better be 

`self.protocol_flags["ENCODING"] = settings.ENCODINGS[0] if settings.ENCODINGS else 'utf-8'`